html/common/config.php

```diff
@@ -54,8 +54,15 @@
                 'secret' => getenv('bCMS__OAUTH__GOOGLESECRET')
             ],
             'scope' => 'https://www.googleapis.com/auth/userinfo.profile https://www.googleapis.com/auth/userinfo.email'
+        ],
+        'SLACK' => [
+            'keys' => [
```

Comment:
Where are these keys used?

Reply:
Line 50 in linkoauth.php - basically the same as google, they are taken from the Adam-rms bot settings on slack